src/libmongoc/src/mongoc/service-gcp.c

```diff
@@ -143,7 +145,7 @@ gcp_access_token_from_gcp_server(gcp_service_account_token *out,
    };
    gcp_request_init(&req, opt_host, opt_port, opt_extra_headers);
 
-   if (!_mongoc_http_send(&req.req, 3 * 1000, false, NULL, &resp, error)) {
+   if (!_mongoc_http_send(&req.req, mlib_duration(3, s), false, NULL, &resp, error)) {
```

Comment:
```suggestion
   if (!_mongoc_http_send(&req.req, socket_timeout_ms, false, NULL, &resp, error)) {
```
Perhaps name this constant as done in `_send_http_request`?